pkg/virt-controller/services/template.go

```diff
@@ -713,6 +713,10 @@ func (t *templateService) renderLaunchManifest(vmi *v1.VirtualMachineInstance, t
 		resources.Requests[key] = value
 	}
 
+	if _, ephemeralStorageDefined := resources.Requests[k8sv1.ResourceEphemeralStorage]; !ephemeralStorageDefined {
+		resources.Requests[k8sv1.ResourceEphemeralStorage] = resource.MustParse("1Mi")
```

Comment:
@stu-gott raised a good point here: if the user requests X MBs of ephemeral-storage then the amount that should be requested is X+1Mb since we should guarantee the user can all of his requested storage.

I'm updating and pushing again.

Reply:
users cannot request anything for container-disk, only for the VMI. so this overhead should be explicitly set for the container-disk.
For the compute container, I don't think we've ever considered users requesting this, but we don't prevent it.
I think you can follow the example of how memory overhead is being added and do something similar with the ephemeral-storage overhead 
https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-controller/services/template.go#L777-L786

Reply:
I think we should increase this to 50Mi to account for unknowns here. storage is cheap, and the repercussions for getting this wrong are high.

Reply:
Thansk @davidvossel . Do you mean 50M to both compute and container disk?

Reply:
> Thansk @davidvossel . Do you mean 50M to both compute and container disk?

Yes, 50Mi for both should be fine.

Reply:
> Thansk @davidvossel . Do you mean 50M to both compute and container disk?

Yes, 50Mi for both should be fine.